07-Customizing-AzSK-for-your-Org/Extending AzSK Module/Readme.md

```diff
@@ -226,9 +227,11 @@ class SubscriptionCore: SVTBase
 	}
   ```  
   
-  4. 	Now you need to prepare the json for the above new control. You can get started by copying the default base json, rename it to <feature>.ext.json. In this case you need to rename it as SubscriptionCore.ext.json. Remove all the other controls except for one and update it with new control details. See additional instructions as '//comments' on each line in the example JSON below. Note: Remove the comments from JSON if you happen to use the below as-is.
+  4. 	For the above a corresponding control in extension json is required. 
+  	Step 1: Copy the default base json, rename it to <feature>.ext.json. 
+	Step 2: Remove all the other controls except for one being added, update it with new control details. See additional instructions as '//comments' on each line in the example JSON below. Note: Remove the comments from JSON if you happen to use the below as-is.
```

Comment:
json has been used in lowercase throughput this page. It'd be better is we use "json" instead of "JSON"